SU-SWS/stanford_profile
# D8CORE-1177: WYSIWYG heading text alignment

# READY FOR REVIEW

# Summary
- Properly aligns headings now that there are character width limits.

# Review By (Date)
- Wednesday, Feb 12th

# Urgency
- Med/High, needs to be fixed before 1.0.0 release.

# Steps to Test

1. Do a complete build of the 1.x branch of acsf-cardinalsites
2. Review the home page and see headings out of alignment 
3. Check out this branch and clear caches
4. Review the alignment of headings in WYSIWYG now work.

# Affected Projects or Products
- D8CORE

# Associated Issues and/or People
- D8CORE
